grakn-engine/src/main/java/ai/grakn/engine/tasks/manager/multiqueue/MultiQueueTaskManager.java

```diff
@@ -61,14 +64,19 @@
 
     public MultiQueueTaskManager(EngineID engineId) {
         this.engineId = engineId;
-        connection = new ZookeeperConnection(client());
-        stateStorage = new TaskStateZookeeperStore(connection);
+        this.zookeeper = new ZookeeperConnection(client());
+
+        if(properties.getPropertyAsBool(USE_ZOOKEEPER_STORAGE)){
```

Comment:
Duplicate Code [1]: No chance of putting this somewhere common? Default method? Abstract class?